py3status/modules/coin_market.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+"""
+Display cryptocurrency data.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module. A message from the site:
+        Please limit requests to no more than 10 per minute.
+        (default 600)
+    format: display format for this module (default '{format_coin}')
+    format_coin: display format for coins
+        (default '[\?color=degraded {name}] ${price_usd:.2f} {percent_change_24h}')
+    format_separator: show separator only if more than one (default ' ')
+    markets: comma separated list of supported markets https://coinmarketcap.com
+        (default ['btc'])
+    request_timeout: time to wait for a response, in seconds (default 5)
+
+Format placeholder:
+    {format_coin} format for cryptocurrency coins
+
+format_coin placeholders:
+    {24h_volume_usd}      eg 1435150000.0
+    {available_supply}    eg 16404825.0
+    {id}                  eg bitcoin
+    {last_updated}        eg 1498135152
+    {market_cap_usd}      eg 44119956596.0
+    {name}                eg Bitcoin
+    {percent_change_1h}   eg -0.17%
+    {percent_change_24h}  eg -1.93%
+    {percent_change_7d}   eg +14.73%
+    {price_btc}           eg 1.0
+    {price_usd}           eg 2689.45
```

Comment:
@ultrabug Got a log? I hadn't ran into this issue with Python `2`/`3`. 

@tobes By using `{price_gbp}` instead of `{price_usd}` -- Here's what `gbp` looks like https://api.coinmarketcap.com/v1/ticker/?convert=gbp&limit=5  -- You'll see that there are `price_gbp` along with `price_usd` too.

I didn't pick the `price_usd`. It came directly from the website. I just take the hand-off approach and pass everything (if possible) from raw data to py3status without touching.

The `format_coin` option should allow the users... 
1) To set their currency inside... instead of new currency option.
2) To set their currency symbol inside... instead of keeping the currency maps.
3) To set their currency formatting inside (ie if they wanted the symbol before or after the `{price_xxx}`... instead of ---- (We have nothing to deal with this).